### ISSUE TITLE
CSV download of a VLD file is out of order

### ISSUE THREAD

**Issue by Captainkirkdawson** [FreeUKGen/FreeCENMigration#1239]
Reported by @geoffj-FUG for HO107_1943.vld downloaded as csv.
Original vld has correct order

E6 U6 S9 C6: P27

**Comment by Captainkirkdawson** [comment]
I was always under the impression that dependants would be delivered in the order of creation. The case found by @geoffj-FUG shows this to be an invalid assumption.  There were 22,000 dependants in this case. Have applied an order criteria to the delivery and at least on my server they are now delivered in the correct order. Have deployed to test3 (Something to file away @AnneV-Learn)

**Comment by geoffj-FUG** [comment]
Kirk
I have downloaded vld file RG12_1918.vld as a CSV file on test3. The rows in the spreadsheet are now in the correct order.
I have made a couple of changes and uploaded the csv as an Upload file. I ran the No POB Warnings report with no problems.
I corrected the couple of entries that came up as Warnings. They were flags that we used to leave in the file in FC1. The other entries with Warnings were associated with the resetting of schedule numbers and jumps in folio numbers when an ED changes. So far so good.
I replaced the csv file on CSVProc. OK
I started Validation and accepted all Warnings. That removed the ED changes and any POB warnings that came up. All OK
I deliberately Incorporated the csv file without removing the vld file. It incorporated. We now have 2 copies of the file that we can search on test3, the vld file and the csv file. 
Why wasn't I stopped from Incorporating the csv file whilst the vld file was still on the sytsem? 
We need a failsafe that if the vld file has not been removed then the csv file cannot be Incorporated.
Geoff
